src/Exceptions/ClientSafeModelNotFoundException.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace Nuwave\Lighthouse\Exceptions;
+
+use GraphQL\Error\ClientAware;
+
+class ClientSafeModelNotFoundException extends \Exception implements ClientAware
```

Comment:
Why not this?

```suggestion
class ClientSafeModelNotFoundException extends ModelNotFoundException implements ClientAware
```

I think it would be better for code that checks for `instanceof ModelNotFoundException`.